packages/ui-inputs/src/input.tsx

```diff
@@ -0,0 +1,221 @@
+import * as React from 'react'
+import { useState } from 'react'
```

Comment:
I added it to make sure the text always will be vertically aligned to the middle. Although the value is incorrect, it should be 32px, since the height is 32px + 2px of borders = 34px the full heigh